### ISSUE TITLE
here have a bug with a htm link 

### ISSUE THREAD

**Issue by AtlantisDe** [DavidAnson/markdownlint#231]
## here have a bug with a htm link 

- here code is ok 
```c#
<a href="http://www.google.com" target="_blank">Google</a>
```

- here code is a bug
- MD033 - Inline HTML
- [bug show this msg   MD033 - Inline HTML](https://github.com/DavidAnson/markdownlint/blob/v0.17.1/doc/Rules.md#md033)
```c#
<a href="http://www.google.com:9999" target="_blank">Google</a>

PS:http://www.google.com:9999 is a link ,too


```

**Comment by DavidAnson** [comment]
I believe both of your examples should be reporting MD033 because both are an example of inline HTML. I’m not sure why they are behaving differently. I will have a look soon. Thank you!

By the way, if you do not want this behavior, you can disable the rule in your configuration.

**Comment by AtlantisDe** [comment]
@DavidAnson Only This code show the msg
```c#
<a href="http://www.google.com:9999" target="_blank">Google</a>
```
